### DISCUSSION TITLE
search engines supress ?

### DISCUSSION THREAD

**Discussion by Quetzaal** [Eloston/ungoogled-chromium#1882] [Q&A] [answered]
hi, thanks a  lot for this ungooggled fork !!

but, i went to the search engine's configuration's panel and there's here some default search engines ... nice suggestion
- but noway to supress them ? do they are some "partners" of the ungoogled aventure ?

thanks a lot

**Comment by networkException** [comment, marked as answer]
Duplicate of https://github.com/Eloston/ungoogled-chromium/issues/1828

**Comment by Quetzaal** [comment]
thanks for your anwser #networkException

it gave me a momentaneous problem... but after reinstall, i went see these search engines, and the option "delete" was available... 
but only until... there were no modification (keyword in my case) of the searchEngine

but the link to #1828 give the solution without to "reinstall" ... then thanks a lot. ;)
